### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dmikurube**
Updating `README.md` for the latest available 0.5.5.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ Versions
 
 | Plugin version | Supported Gradle versions | Method to manipulate `pom.xml`            |
 | -------------- | ------------------------- | ----------------------------------------- |
-| v0.4.2         | Gradle 6                  | Done by additional Gradle `configuration` |
-| v0.5.3         | Gradle 6 & 7              | Done by `pom.withXml`                     |
+| v0.4.5         | Gradle 6                  | Done by additional Gradle `configuration` |
+| v0.5.5         | Gradle 6 & 7              | Done by `pom.withXml`                     |
+| v0.6.0 (TBA)   | Gradle 7.6                | Done by `pom.withXml`                     |
```
Comment (hiroyuki-sato) [inline README.md:11]: minor 7.6.1?

**Review by hiroyuki-sato — APPROVED** [review]
LGTM👍

File: README.md (hunk 1 above)
Comment (dmikurube) [reply README.md:11]: They should be basically "compatible", then let's keep 7.6 here. (It's general that newer is better in a compatible range.)

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**